webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -674,7 +674,7 @@ export default class PluginManagement extends AdminSettings<Props, State> {
                         values={{
                             link: (msg: React.ReactNode) => (
                                 <ExternalLink
-                                    href='https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta/'
+                                    href='https://mattermost.com/pl/default-beta-plugins/'
```

Comment:
Why "beta"?

Reply:
Good question, @wiggin77. That's an existing permalink. @emdecr - Can we update this permalink to remove the "beta" part, please?

Reply:
The pl resolves to `https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/` which has no `beta` in the url.

Reply:
@cwarnermm I can definitely make a switch - to clarify we want a `/pl/` link for `https://mattermost.com/pl/default-plugins/` that goes to `https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/`?

Reply:
@cwarnermm We have an existing `/pl/` for that one with the project we're doing with Felipe and it goes to https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta 😬 

Maybe `/pl/using-and-managing-plugins`?

Reply:
oh! We're talking about the same destination and we already have a permalink! I think I just used the wrong one in this PR. Sounds like we can safely mark the beta version as a duplicate.

Reply:
Corrected via this PR.